image-swipe.d.ts

```diff
@@ -17,6 +17,11 @@ import { EventData } from "data/observable";
 import { CoercibleProperty, Property } from "ui/core/view";
 import { ItemsSource } from "ui/list-picker";
 import { ScrollView } from "ui/scroll-view";
+import { Cache } from "ui/image-cache";
+
+export class ImageSwipeBase {
```

Comment:
Here are the issues I've submitted for both android and ios runtime:
https://github.com/NativeScript/android-runtime/issues/783
https://github.com/NativeScript/ios-runtime/issues/773